ingestion-beam/src/main/java/com/mozilla/telemetry/contextualservices/ParseReportingUrl.java

```diff
@@ -87,40 +99,104 @@ public Result<PCollection<PubsubMessage>, PubsubMessage> expand(
           }
 
           Map<String, String> attributes = new HashMap<>(message.getAttributeMap());
+          String namespace = Objects
+              .requireNonNull(message.getAttribute(Attribute.DOCUMENT_NAMESPACE));
```

Comment:
This will throw `NullPointerException` which isn't currently caught, so will cause the whole pipeline to fail. We may want to throw an `IllegalAttributeException` instead.